atomics/T1550.002/T1550.002.yaml

```diff
@@ -21,9 +21,26 @@ atomic_tests:
       description: domain
       type: string
       default: atomic.local
+    mimikatz_path:
+      description: mimikatz windows executable
+      type: Path
+      default: '%tmp%\mimikatz\x64\mimikatz.exe'
```

Comment:
Because you're using PowerShell below, I'd recommend using a string format that it will implicitly understand without needing to resort to calling cmd.exe to resolve a string. Example:

```
default: $Env:\Temp\mimikatz\x64\mimikatz.exe
```